Fomo/TripEvent.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  TripEvent.swift
+//  Fomo
+//
+//  Created by Christian Deonier on 2/29/16.
+//  Copyright © 2016 TeamAwesome. All rights reserved.
+//
+
+import UIKit
+
+class TripEvent: NSObject {
+    
+    enum Vote: Int {
+        case Dislike = 1, Neutral, Like
+    }
+
+    var id: Int?
+    var attraction: Attraction?
+    var startTime: NSDate?
+    var endTime: NSDate?
```

Comment:
in lieu of startTime and endTime, would it be simpler to do date (day) + an enum for time of day (morning/afternoon/evening)?